udp-ballerina/tests/connectionless_client_tests.bal

```diff
@@ -27,11 +27,19 @@ function setup() {
 @test:Config {
 }
 function testClientEcho() {
-    Client socketClient = new;
+    Client socketClient = new ("localhost",2000);
     string msg = "Hello Ballerina echo";
-    var sendResult = socketClient->sendTo(msg.toBytes(), {host: "localhost", port: 48829});
-    if (sendResult is int) {
-        log:print("Number of bytes written: " + sendResult.toString());
+    Datagram datagram = {
+        remoteAddress : {
+            host : "localhost",
+            port : 48829
+        },
+        data : msg.toBytes()
+    };
+
+    var sendResult = socketClient->send(datagram);
+    if (sendResult is ()) {
+        log:print("datagram sent to remote address");
```

Comment:
```suggestion
        log:print("Datagram was sent to remote address.");
```
Make sure the grammatical correctness in printed messages :)
BTW is it the `address` or the `host` we have to mention? As we have provided the `host` parameter, I think it should be the `host` instead of the `address`.

Reply:
updated